java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java

```diff
@@ -57,72 +61,276 @@ public TaskManagerClient(String endpoint) {
+     * @throws Exception
+     */
+    public void runBatchSql(String sql, String output_path, HashMap<String, String> conf, String default_db) throws Exception {
+        TaskManager.RunBatchSqlRequest request = TaskManager.RunBatchSqlRequest.newBuilder()
+                .setSql(sql)
+                .setOutputPath(output_path)
+                .putAllConf(conf)
+                .setDefaultDb(default_db)
+                .build();
+        TaskManager.ShowJobResponse response = taskManagerInterface.RunBatchSql(request);
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+    }
+    /**
+     * Run batch sql statements and display the results without default_db.
+     *
+     * @param sql query sql string.
+     * @throws Exception
+     */
+    public String runBatchAndShow(String sql) throws Exception {
+        String Msg = runBatchAndShow(sql,new HashMap<String ,String>(),"");
+        return Msg;
+    }
+    /**
+     * Run batch sql statements and display the results.
+     *
+     * @param sql query sql string.
+     * @param conf configure.
+     * @param default_db default database.
+     * @throws Exception
+     */
+    public String runBatchAndShow(String sql, HashMap<String, String> conf, String default_db) throws Exception {
+        TaskManager.RunBatchAndShowRequest request = TaskManager.RunBatchAndShowRequest.newBuilder()
+                .setSql(sql)
+                .putAllConf(conf)
+                .setDefaultDb(default_db)
+                .build();
+        TaskManager.ShowJobResponse response = taskManagerInterface.RunBatchAndShow(request);
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+        return response.getMsg();
+    }
+    /**
+     * Show the job rely on id.
+     *
+     * @param id job's id.
+     * @return the JobInfo object.
+     * @throws Exception
+     */
+    public TaskManager.JobInfo showJob(int id) throws Exception {
+        TaskManager.ShowJobRequest request = TaskManager.ShowJobRequest.newBuilder()
+                .setId(id)
+                .build();
+        TaskManager.ShowJobResponse response = taskManagerInterface.ShowJob(request);
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+        return response.getJob();
+    }
+    /**
+     * Import online data without default_db.
+     *
+     * @param sql query sql string.
+     * @throws Exception
+     */
+    public String importOnlineData(String sql) throws Exception {
+        String Msg = importOnlineData(sql, new HashMap<String, String>(), "");
```

Comment:
Return `importOnlineData(sql, new HashMap<String, String>(), "");` directly and do not need to use new variable `Msg`.

Reply:
I get it